node/src/accountant/db_access_objects/receivable_dao.rs

```diff
@@ -705,8 +809,10 @@ mod tests {
             },
         ];
 
-        subject.more_money_received(payment_time, transactions);
+        let mut txn = subject.more_money_received(payment_time, &transactions);
 
+        txn.commit().unwrap();
+        drop(txn);
```

Comment:
I don't see a return type for `more_money_received()`. I'm confused how did you end up receiving the `txn` out of it and then managed to call functions on it, and ultimately drop it.

Secondly, I don't know what's the purpose of doing this exercise.